src/atoms/second_order_cone/quadform.jl

```diff
@@ -1,6 +1,6 @@
 export quadform
 
-function is_symmetric(A::Matrix)
+function is_symmetric(A::AbstractArray)
   TOLERANCE = 1e-6
   return all(A - A' .< TOLERANCE)
```

Comment:
it's probably clearer with abs, but it doesn't matter: if A[i,j] - A'[i,j]
= A[i,j] - A[j,i] is large and negative, then A[j,i] - A'[j,i] = A[j,i] -
A[i,j] is large and positive.

On Sun, Jul 10, 2016 at 10:50 AM, Tony Kelman <notifications@github.com>
wrote:

> In src/atoms/second_order_cone/quadform.jl
> <https://github.com/JuliaOpt/Convex.jl/pull/149#discussion_r70183086>:
>
> > @@ -1,6 +1,6 @@
> >  export quadform
> >
> > -function is_symmetric(A::Matrix)
> > +function is_symmetric(A::AbstractArray)
> >    TOLERANCE = 1e-6
> >    return all(A - A' .< TOLERANCE)
>
> this should probably be abs(A - A')
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaOpt/Convex.jl/pull/149/files/10f5913c6183410633782fd5aa3faf30155e1109#r70183086>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAyp9F4w-Ki_WpmyxU8YPhRLCPQ-BpxCks5qUQamgaJpZM4JIeLH>
> .
>



-- 
Madeleine Udell
Assistant Professor, Operations Research and Information Engineering
Cornell University
*https://courses2.cit.cornell.edu/mru8
<https://courses2.cit.cornell.edu/mru8>*
(415) 729-4115